packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -228,6 +235,19 @@ function InserterBlockList( {
 					);
 				} ) }
 
+			{ ! hasChildItems && uncategorizedItems.length && (
+				<InserterPanel
+					className="block-editor-inserter__uncategorized-blocks-panel"
+					title={ __( 'Uncategorized' ) }
```

Comment:
For uncategorized patterns, I've used "All" as a label. "Uncategorized" is too technical and can confuse users. "All" is inaccurate. Not sure what's best.

Reply:
> For uncategorized patterns, I've used "All" as a label. "Uncategorized" is too technical and can confuse users. "All" is inaccurate. Not sure what's best.

Well, there is the precedent of the "Uncategorized" default post category 🤷

Reply:
If we don't just go with "Uncategorized", then I think "Other" would work.

Reply:
I think it should be fine to stick with "Uncategorized"? I guess we can at least see what feedback there is, if any.